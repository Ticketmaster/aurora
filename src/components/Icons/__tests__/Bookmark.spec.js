/* eslint-disable import/no-extraneous-dependencies */
import React from "react";
import renderer from "react-test-renderer";

import { BookmarkIcon } from "../";

describe("BookmarkIcon", () => {
  const children = <title>Bookmark Icon</title>;

  it("renders correctly", () => {
    const component = renderer.create(<BookmarkIcon>{children}</BookmarkIcon>);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
