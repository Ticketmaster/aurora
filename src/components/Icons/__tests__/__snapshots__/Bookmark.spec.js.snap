// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookmarkIcon renders correctly 1`] = `
<svg
  height="16"
  viewBox="0 0 16 16"
  width="16"
  xmlns="http://www.w3.org/2000/svg"
>
  <title>
    Bookmark Icon
  </title>
  <path
    d="M13 1H3v13.29l3.957-2.229a2 2 0 0 1 1.946-.01L13 14.309V1zM3 0h10a1 1 0 0 1 1 1v15l-5.58-3.072a1 1 0 0 0-.973.005L2 16V1a1 1 0 0 1 1-1z"
    fill="#262626"
    fillOpacity=".65"
    fillRule="nonzero"
  />
</svg>
`;
